cmd/soroban-cli/src/wasm.rs

```diff
@@ -42,7 +42,7 @@ pub enum Error {
     #[error(transparent)]
     Rpc(#[from] soroban_rpc::Error),
     #[error("unexpected contract data {0:?}")]
-    UnexpectedContractToken(ContractDataEntry),
+    UnexpectedContractToken(Box<ContractDataEntry>),
```

Comment:
This change requested by the linter is okay, but it's worth noting it's an optimisation, and not really required to Box it, so we could also simply silence the lint on this type since we know it's okay for the enum to have a variety of size variants.